src/test/java/ch/admin/bag/covidcertificate/TestModelProvider.java

```diff
@@ -253,4 +260,40 @@ public static ExceptionalCertificateDataDto getExceptionalCertificateDataDto() {
     public static CovidCertificateAddressDto getCovidCertificateAddressDto() {
         return new CovidCertificateAddressDto("street 12", 2500, "Bern", "BE");
     }
+
+    public static VaccinationCertificateConversionRequestDto getVaccinationCertificateConversionRequestDto() {
+        return new VaccinationCertificateConversionRequestDto(
+                ConversionReason.VACCINATION_CONVERSION,
+                new VaccinationCertificateHcertDecodedDto(
+                        "1.3.0",
+                        new CertificatePersonDto(
+                                new CertificatePersonNameDto(
+                                        "Düsentrieb",
+                                        "DUESENTRIEB",
+                                        "Daniel",
+                                        "DANIEL"),
+                                LocalDate.now().minusYears(20).minusMonths(2).minusDays(4).toString()
+                        ),
+                        List.of(getVaccinationCertificateHcertDecodedDataDto())
+                )
+        );
+    }
+
+    public static VaccinationCertificateHcertDecodedDataDto getVaccinationCertificateHcertDecodedDataDto() {
+        return new VaccinationCertificateHcertDecodedDataDto(
+                "840539006",
+                "1119349007",
+                "EU/1/20/1507",
+                "ORG-100031184",
+                Integer.valueOf(2),
+                Integer.valueOf(2),
+                LocalDate.now().minusDays(10),
+                "CH",
+                "Bundesamt für Gesundheit (BAG)",
+                createUVCI());
+    }
+
+    public static String createUVCI() {
```

Comment:
Zur Info: eine createUVCI Methode existiert bereits in FixtureCustomization. Man könnte auch diese wiederverwenden anstelle eine neue zu machen.

Reply:
Stimmt, von dort hab ich sie. Da der TestModelProvider ohne Fixture arbeitet, hab ich sie nicht verwendet. Ich passe es dennoch an.